src/lib/one-graph/netlify-graph.js

```diff
@@ -0,0 +1,705 @@
+const { randomUUID } = require('crypto')
+const fs = require('fs')
```

Comment:
You can use the promise version if you want to :)

```suggestion
const { mkdir } = require('fs').promises
```

Reply:
I don't see any benefit to it, I prefer to use sync for these operations.